conf.d/globalblacklist.conf

```diff
@@ -18123,6 +18123,22 @@ geo $ratelimited {
 # END SECTION 4 - ACTIVATE BLOCKER FUNCTIONS
 # ==========================================
 
+
```

Comment:
Make this reference the rules in bad-referrer-words.conf, it also cannot be added here it needs to be added in the two  master template files 
- https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/blob/master/.dev-tools/globalblacklist.template
- https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/blob/master/.dev-tools/globalblacklist.template